x-pack/plugin/security/lib/nimbus-jose-jwt-modified/src/main/java/com/nimbusds/jose/util/JSONObjectUtils.java

```diff
@@ -192,6 +193,16 @@ public static Base64URL getBase64URL(final Map<String, Object> o, final String k
         }
     }
 
+    public static Date getEpochSecondAsDate(final Map<String, Object> o, final String key) throws ParseException {
```

Comment:
This method was added since we last upgraded: https://bitbucket.org/connect2id/nimbus-jose-jwt/commits/bd59d30bfc33c6eb8ec9514f67a4261d95c894c4#Lsrc/main/java/com/nimbusds/jose/util/JSONObjectUtils.javaT519